client/state/mailchimp/settings/actions.js

```diff
@@ -4,7 +4,6 @@ import {
 	MAILCHIMP_SETTINGS_RECEIVE,
 	MAILCHIMP_SETTINGS_UPDATE,
 	MAILCHIMP_SETTINGS_UPDATE_SUCCESS,
-	MAILCHIMP_SETTINGS_UPDATE_FAILURE,
```

Comment:
Two opportunities for more cleanup:
- the `receiveSettings` action creator in this module is not used anywhere. Either remove it, or use it in the dispatches that construct a `MAILCHIMP_SETTINGS_RECEIVE` action explicitly, without a creator.
- the `MAILCHIMP_SETTINGS_UPDATE_SUCCESS` is identical to `MAILCHIMP_SETTINGS_RECEIVE`, the reducer handles them the same way. That's an opportunity to remove `MAILCHIMP_SETTINGS_UPDATE_SUCCESS`.

Reply:
> * the `receiveSettings` action creator in this module is not used anywhere. Either remove it, or use it in the dispatches that construct a `MAILCHIMP_SETTINGS_RECEIVE` action explicitly, without a creator.

This is making me think it'd be valuable to also have checks for use of exported action creators elsewhere in the application. I've now seen several examples of action creators that exist but are never used.